src/components/Card.svelte

```diff
@@ -17,6 +17,11 @@
   let pinned: boolean;
   $: pinned = $settings.pinnedFiles.includes(file.path);
 
+  // Compute style based on settings
+  $: cardStyle = $settings.maxCardHeight
+    ? `max-height: ${$settings.maxCardHeight}px; overflow: hidden; text-overflow: ellipsis;`
+    : "";
+
```

Comment:
This seems to be from another PR / feature.

Reply:
Yes, my bad, forgot to remove that. Fixed.